Foundation/NSCalendar.swift

```diff
@@ -147,6 +147,26 @@ public class NSCalendar : NSObject, NSCopying, NSSecureCoding {
         }
     }
     
+    public override var hash: Int {
+        get {
+            return Int(CFHash(_cfObject))
```

Comment:
I still think these need to be bit pattern for the same reason that NSCFString's hash needed it